src/assignment/views/AssignmentEdit.tsx

```diff
@@ -469,15 +467,37 @@ const AssignmentEdit: FunctionComponent<AssignmentEditProps> = ({
 		}
 	};
 
-	const duplicateAssignment = async (title: string) => {
-		const duplicatedAssigment = await insertDuplicateAssignment(
-			triggerAssignmentInsert,
-			title,
-			initialAssignment
+	const triggerCollectionCopy = async (
+		contentLabel: string
+	): Promise<Avo.Collection.Collection | null> => {
+		const isCollection = contentLabel === 'COLLECTIE';
+		const isOwnedByCurrentUser = !(
```

Comment:
deze check moet enkel gebeuren bij het aanmaken van een opdracht. 
Bij het kopiëren van een opdracht moet er altijd een kopie gemaakt worden van de collectie zelfs als de gebruiker eigenaar is van de collectie die gelinked is aan de source assignment